pkgs/applications/blockchains/bitcoin/default.nix

```diff
@@ -11,7 +11,6 @@
   wrapQtAppsHook ? null,
   boost,
   libevent,
-  miniupnpc,
```

Comment:
has this been dropped from requirements?

Reply:
yeah, see https://github.com/bitcoin-core/bitcoin-devwiki/wiki/29.0-Release-Notes-draft#p2p-and-network-changes

Reply:
Adding this to the commit message.